### PR TITLE
Add slovak/SK language support

### PR DESCRIPTION
**Pull request by sbocinec**
* Add slovak (SK) language translations
* Add support for SK URL routes

### CODE REVIEW

File: app/locales/sk.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  " if you can't get it to work.": " ak sa Vám to nedarí sprevádzkovať.",
+  "Before proceeding please note down the following link": "Pred pokračovaním si prosím uložte nasledujúci link",
+  "Save the following link somewhere, or bookmark it in your browser.": "Uložte si tento odkaz alebo si ho pridajte do záložiek.",
+  "That way you can update your report if your health condition changes.": "Touto cestou viete aktualizovať Váš report v prípade zmeny zdravotného stavu.",
+  "I've noted down the link.": "Zapísal som si tento odkaz.",
+  "Proceed": "Pokračovať",
+  "The link has been copied": "Odkaz bol skopírovaný.",
+  "Could not copy the link, please copy it yourself": "Odkaz sa nedá skopírovať, skopírujte si ho manuálne",
+  "Contributors": "Prispievatelia",
+  "This website is made with help from volunteers. In no particular order:": "Táto stránka je vytvorená s pomocou dobrovoľných prispievateľov. Bez konkrétneho poradia:"
```
Comment (fossecode) [inline app/locales/sk.json]: Missing comma here

File: app/locales/sk.json [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  "https://who.org": "https://www.uvzsr.sk",
+  "To WHO.org": "Na uvzsr.sk",
+  "Developed by": "Vytvorené spol.",
+  "BustByte AS": "BustByte AS",
+  "and": "a",
+  "volunteers": "dobrovoľníkmi",
+  "Privacy Policy": "Ochrana osobných údajov",
+  "Privacy": "Súkromie",
+  "Thank you for your contribution!": "Ďakujeme za Váš príspevok!",
+  "The government has now launched their own self-report system, please use that instead.": "Štát už zriadil vlastný systém na samo ohlasovane stavu, prosím použite radšej tento."
```
Comment (fossecode) [inline app/locales/sk.json]: Missing comma

File: app/locales/sk.json (hunk 1 above)
Comment (sbocinec) [reply app/locales/sk.json]: thank you, will push in a batch fixed together with others

File: app/locales/sk.json (hunk 2 above)
Comment (sbocinec) [reply app/locales/sk.json]: thank you, will push in a batch fixed together with others

File: app/domain/urls.ts [hunk 3]
```diff
@@ -45,6 +45,16 @@ const localeAwareUrls: Urls = {
...
     statistics: '/statistics'
+  },
+  sk: {
+    submitReport: '/',
+    profile: '/zdravotny-stav',
+    privacyPolicy: '/ochrana-udajov',
+    map: '/mapa',
+    contributors: '/prispievatelia',
+    api: '/api',
+      apiDocs: '/api-docs',
```
Comment (fossecode) [inline app/domain/urls.ts]: Not critical, but the indentation is wrong

**Comment by fossecode** [comment]
Fantastic! :)

File: app/domain/urls.ts (hunk 3 above)
Comment (sbocinec) [reply app/domain/urls.ts]: yeah, nice catch, thank you!

**Comment by michaelmcmillan** [comment]
@all-contributors please add @sbocinec for translating the site into slavic

**Comment by fossecode** [comment]
You could also add yourself as codeowner of the sk translations: https://github.com/BustByte/coronastatus/blob/master/CODEOWNERS#L5

**Review by fossecode — APPROVED** [review]

**Comment by sbocinec** [comment]
One more thing - added Slovak wordlist https://github.com/BustByte/coronastatus/pull/231/commits/70726ae62f096ecd675b320a7f52ca8a1c4fad7e - Could not find any good one, so took the [english BTC](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt), translated using Google translate and did some normalizations (diacritic removal/transliteration, lowercase,,...). Looks better than nothing

**Comment by adriaandotcom** [comment]
@sbocinec is this ready to merge?

**Comment by sbocinec** [comment]
> @sbocinec is this ready to merge?

It will require few more fixes + adding the privacy policy, however, will create a new PR for those.

**Pull request merged**